blockchain/for-validators-nominators-and-stakers/resync.md

```diff
@@ -0,0 +1,47 @@
+# Resyncing node
+
+To resync your Liberland Node, you need to:
+1. stop the node
```

Comment:
Missing step. Include stopping validating in blockchain.liberland.org or polkadotjs and waiting until you are not in active validator set

Reply:
add this step to other places where applicable

Reply:
Done. Other places don't need it (this is the only instruction that causes prolonged downtime of the node).